### DISCUSSION TITLE
Linux - Vanilla Setup

### DISCUSSION THREAD

**Discussion by giscus** [Uuvana-Studios/longvinter-server-documentation#1] [Announcements]
# Linux - Vanilla Setup

Longvinter Server Documentation for Server Administrators

https://docs-server.longvinter.com/linux/setup-vanilla/#backing-up-your-saves

**Comment by JustKing009** [comment]
I get this message in my Terminal, when i start my longvinter server.
$ sh /home/steam/longvinter-linux-server/LongvinterServer.sh
4.26.0-0+++UE4+Release-4.26 522 0
Disabling core dumps.
Preparing to exit.Shutting down and abandoning module SandboxFile (6)Shutting down and abandoning module PakFile (4)Shutting down and abandoning module RSA (3)Exiting.Exiting abnormally (error code: 1)

**Reply by senlsx** [reply]
I get it too，so how can we fix it？

**Reply by jskiweb** [reply]
Same Thing for me

**Reply by jskiweb** [reply]
Did someone managed to fix it ?

**Reply by laskdjlaskdj12** [reply]
No Same in ubuntu 20.04

**Reply by senlsx** [reply]
I open a issue on the github，but still not anyone reply me

**Reply by Rytiggy** [reply]
So on their discord someone mentioned this works:

You will need to install lfs first  curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
Then run these cmds in the server dir. 
git lfs fetch --all
git lfs pull


But after running those two cmds the server launched with out problems

Hope this helps.

**Reply by dalekcoffee** [reply]
Agree with Rytiggy, 

the problem with this guide is that when a newer user runs "sudo apt install git git-lfs steamcmd" and sees it does not work, they move on to the 64 bit instructions.

Git is already installed, but git-lfts is skipped

Now I was more interested in where I messed up so I went through the process all over again more carefully
If  you already installed, try what he said

If you wanna say F it and start over, whenever you get to that command just run it one package at a time instead of all 3

sudo apt install git
sudo apt install git-lfs
sudo apt install steamcmd (wont work on 64bit, move on to the 64bit instructions)

**Comment by HHNebula** [comment]
What does the setting below mean in the Game.ini file and how should I set it?

CoopPlay=
CoopSpawn=
